kickoff/config.py

```diff
@@ -8,8 +8,8 @@
 ESR_NEXT = "60"
 JSON_FORMAT_VERSION = "1.0"
 JSON_FORMAT_L10N_VERSION = "1.0"
-IOS_BETA_VERSION = "9.1"
-IOS_VERSION = "9.0"
+IOS_BETA_VERSION = ""
+IOS_VERSION = "12.0"
```

Comment:
I see on https://itunes.apple.com/us/app/firefox-web-browser/id989804926 we shipped 12.1 on June 8th. I couldn't find the beta page, though.